src/models/createNewHouseModel.js

```diff
@@ -0,0 +1,103 @@
+const mongoose = require('mongoose')
+
+const housesSchema = new mongoose.Schema({
+    owner: {
+        type: mongoose.Schema.Types.ObjectId,
+        required: true
+    },
+    address:{
+        country:{
+            type: String,
+            required: true
+        },
+        region:{
+            type: String
+        },
+        town:{
+            type: String,
+            required: true
+        },
+        district: {
+            type:String,
+        },
+        street:{
+            type: String,
+            required: true
+        },
+        houseNr: {
+            type: String,
+            required: true
+        }
+    },
+    houseType: {
+        type: String,
+        required: true
+    },
+    attributes:{
+        nrOfRooms:{
+            type: Number,
+            required: true
+        },
+        nrOfBathrooms:{
+            type: Number,
+            required: true
+        },
+        floors:{
+            type: Number,
+            required: true
+        },
+        heating:{
+            type: String,
+            required: true
+        }
+    },
+    OtherFeatures:{
+        type: Array,
+        default: []
+    },
+    constructionDate:{
+        type: Number,
+    },
+    area: {
+        type: Number,
+        require: true
+    },
+    price: {
+        type: Number,
+        required: true
+    },
+    meta: {
+        houseId: {
```

Comment:
I think houses will need a not nested `_id` to make them more easily referable for other schemas.